src/test/cucumber/mnemonics.feature

```diff
@@ -38,22 +38,41 @@ Feature: Mnemonics
     And I see "grails: 2.1.0"
     And I see "groovy: 2.0.5"
 
-  Scenario: Shortcut for displaying outdated Candidate Version in use
+  Scenario: Shortcut for displaying upgradable Candidate Version in use
+    Given the candidate "grails" version "1.3.9" is already installed and default
+    And the default "grails" version is "2.4.4"
+    And the system is bootstrapped
+    When I enter "sdk g grails"
+    Then I see "Upgradable:"
+    And I see "grails (1.3.9 < 2.4.4)"
+
+  Scenario: Shortcut for displaying upgradable Candidate Versions
```

Comment:
I would probably drop this scenario as it doesn't add much to what the one above provides.